### PR TITLE
Rework questions

### PR DESCRIPTION
**Pull request by JasonXu314**

### CODE REVIEW

**Review by Cire15 — CHANGES_REQUESTED** [review]
The changes are good, just a few things to comply with MLH.

File: pages/application.tsx [hunk 1]
```diff
@@ -84,23 +350,56 @@ const Application: NextPage = () => {
...
+										{...form.getInputProps('age')}
+									/>
+									<NativeSelect
+										required
+										label="In which country do you currently reside?"
+										data={countries}
+										{...form.getInputProps('country')}
+									/>
+									<Group spacing={0}>
+										<TextInput
```
[the remark this answers is not in the document]
Comment (JasonXu314) [reply pages/application.tsx:366]: the dropdown was so big that opening it caused it to flash black, so we changed it to a free input with the note

[the remark this answers is not in the document]
Comment (Cire15) [reply pages/application.tsx:366]: if they're cool with it, I'm cool with it

**Review by Cire15 — APPROVED** [review]
👍

**Pull request merged**